storage/index.js

```diff
@@ -1 +1,15 @@
 // JavaScript source code
+var server = require('./app.js');
```

Comment:
Cannot require app.js inside a moudle. Storage is module which is to be required inside app and config to be passed to it.